clients/include/near.h

```diff
@@ -87,4 +87,39 @@ HIPBLAS_CLANG_STATIC constexpr double error_tolerance = 0.0;
 template <>
 HIPBLAS_CLANG_STATIC constexpr double error_tolerance<hipblasHalf> = 0.000061035;
 
+template <class Tc, class Ti, class To>
+static constexpr double sum_error_tolerance_for_gfx11 = 0.0;
+
+template <>
+HIPBLAS_CLANG_STATIC constexpr double sum_error_tolerance_for_gfx11<float, hipblasBfloat16, float>
+    = 1 / 10000.0;
+
+template <>
+HIPBLAS_CLANG_STATIC constexpr double
+    sum_error_tolerance_for_gfx11<float, hipblasBfloat16, hipblasBfloat16>
+    = 1 / 10000.0;
+
+template <>
+HIPBLAS_CLANG_STATIC constexpr double sum_error_tolerance_for_gfx11<float, hipblasHalf, float>
+    = 1 / 10000.0;
+
+template <>
+HIPBLAS_CLANG_STATIC constexpr double sum_error_tolerance_for_gfx11<float, hipblasHalf, hipblasHalf>
+    = 1 / 10000.0;
+
+template <>
+HIPBLAS_CLANG_STATIC constexpr double
+    sum_error_tolerance_for_gfx11<hipblasHalf, hipblasHalf, hipblasHalf>
+    = 1 / 10000.0;
```

Comment:
Your build may not be using the new assembly instructions yet, it is in progress.  I would test with navi32 to see the future, or could go with what rocblas uses.  Maybe your tests are all small enough K ?

Reply:
I built in the docker container in the ticket and it failed using unit_check() and passed with these tolerances. I don't mind changing it to match rocBLAS if we're concerned with future changes affecting this, though, just seemed excessive to use 1/10.0 when 1/10000.0 seemed to work.

Reply:
If it passes in the docker I would favor your tighter tolerance. You could add a comment that if k is large the tolerance would need to increase.

Reply:
Okay K is so small it isn't really doing any reduction, but still would test on gfx1101

Reply:
Looking at the tests, it looks like the largest k is 1027, (M=925, N=1026, K=1027).
Tests all passed on gfx1101 just now.